en/docs/fragments/ask-password.md

```diff
@@ -0,0 +1,96 @@
+1.  Log in to the Management Console (`https://<IS_HOST>:<PORT>/carbon`).
+
+2.  Click **Resident** under **Identity Providers** on the **Main** tab and expand the **Account Management Policies** tab.
```

Comment:
Main > **Identity**

Reply:
Account Management Policies section has been renamed and the sections inside it also has changed.